### PR TITLE
revert ledger design commit

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 1]
```diff
@@ -226,28 +222,44 @@ <h3 class="ledgerHead">
...
+                                                                    (noResultsClicked)="showQuickAccountModal()"
+                                                                    [showClear]="false" [multiple]="false"
+                                                                    [ItemHeight]="67"
+                                                                    [useInBuiltFilterForFlattenAc]="true">
                                                                     <ng-template #optionTemplate let-option="option">
                                                                         <ng-container *ngIf="!option.additional?.stock">
-                                                                            <a href="javascript:void(0)" class="list-item" style="border-bottom: 1px solid #ccc;">
+                                                                            <a href="javascript:void(0)"
+                                                                                class="list-item"
+                                                                                style="border-bottom: 1px solid #ccc;">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/ledger.component.html:262]: avoid inline css

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 2]
```diff
@@ -256,34 +268,50 @@ <h3 class="ledgerHead">
...
+                                                                <ul class="no-result-template"
+                                                                    *ngIf="lc.noAccountChosenForNewEntry">
                                                                     <li>Create New Account</li>
                                                                 </ul>
                                                             </td>
 
-                                                            <td class="col-xs-4 col-md-3 col-lg-3  pos-rel">
-                                                                <input class="form-control alR" type="text" decimalDigitsDirective [DecimalPlaces]="2" [(ngModel)]="txn.amount" (input)="needToReCalculate.next(true)" (click)="blankLedgerAmountClick()">
+                                                            <td class="col-xs-3 pos-rel">
+                                                                <input class="form-control alR" type="text"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/ledger.component.html:293]: are we using input type tel for amount field?

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 3]
```diff
@@ -293,30 +321,43 @@ <h3 class="ledgerHead">
...
+                                                                    (noResultsClicked)="showQuickAccountModal()"
+                                                                    [multiple]="false" [ItemHeight]="67"
+                                                                    [showClear]="false"
+                                                                    [useInBuiltFilterForFlattenAc]="true">
                                                                     <ng-template #optionTemplate let-option="option">
                                                                         <ng-container *ngIf="!option.additional?.stock">
-                                                                            <a href="javascript:void(0)" class="list-item" style="border-bottom: 1px solid #ccc;">
+                                                                            <a href="javascript:void(0)"
+                                                                                class="list-item"
+                                                                                style="border-bottom: 1px solid #ccc;">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/ledger.component.html:360]: avoid inline css

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:262]: ye phle ka code h ravindr ji mene revert kiya h vo lines uper niche hogyi

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:360]: ye phle ka code h ravindr ji mene revert kiya h vo lines uper niche hogyi

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 1 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:262]: Yes i know, can we fix this as improvement?

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 3 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:360]: Yes i know, can we fix this as improvement?

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:360]: Yes we can this but later abhi prod per jana h ye , ledger bada kaam hai timelgega

**Pull request merged**